scripts/sync_test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# We consider a good sync if the latest block synced is OLDEST_ACCEPTABLE seconds old
+OLDEST_ACCEPTABLE=30
+
+if [ -z "$DATADIR" ]; then
+  echo "Set DATADIR to the desired datadir folder"
+  exit 3
+fi
+
+if [ -z "$MODE" ]; then
+  echo "Set MODE to the sync mode"
+  exit 3
+fi
+LOGFILE=/tmp/sync_test.log
+
+# Do the sync
+echo "Running geth sync"
+build/bin/geth --datadir $DATADIR --syncmode $MODE --exitwhensynced > $LOGFILE 2>&1
+
+
+# Now check what the latest block is
+ATTEMPTS=5
+RETRY_SLEEP=2
+# We attempt to check it several times since, sometimes the command
+# fails with "No peers available"
+for ATTEMPT in $(seq 1 $ATTEMPTS); do
+	echo "Attempt $ATTEMPT/$ATTEMPTS of getting the latest block timestamp" 
+	MARK=`date +%s`
+	LATEST=`build/bin/geth --datadir $DATADIR --verbosity 0 console --syncmode $MODE --exec 'eth.getBlock("latest").timestamp'`
```

Comment:
no need to do use `--syncmode` here

Reply:
If I don't use it it reports a different block. It seems it reports the latest block downloaded in the database for the specified sync mode